libcontainer/rootfs_linux.go

```diff
@@ -377,25 +377,35 @@ func mountToRootfs(m *configs.Mount, c *mountConfig) error {
 	rootfs := c.root
 	mountLabel := c.label
 	mountFd := c.fd
-	dest, err := securejoin.SecureJoin(rootfs, m.Destination)
-	if err != nil {
-		return err
-	}
 
 	switch m.Device {
 	case "proc", "sysfs":
 		// If the destination already exists and is not a directory, we bail
-		// out This is to avoid mounting through a symlink or similar -- which
+		// out. This is to avoid mounting through a symlink or similar -- which
 		// has been a "fun" attack scenario in the past.
 		// TODO: This won't be necessary once we switch to libpathrs and we can
 		//       stop all of these symlink-exchange attacks.
+		dest := m.Destination
```

Comment:
Do we need to `filepath.Clean()` before checking the prefix?

Reply:
Probably `dest := filepath.Clean("/"+m.Destination)` makes the most sense.

Reply:
In the light of that, I guess, we need to finish what we started in commits 1f1e91b1a and b31a9340f. Before doing that, I guess, we could not use `filepath.Clean("/"+m.Destination)`.

Reply:
And once we'll finish it, it won't be necessary to prepend `"/"`. So, I guess, bare `filepath.Clean` is enough.